### PR TITLE
Correlation Key property added

### PR DESCRIPTION
**Pull request by ghost**
Added additional property - CorrelationKey to alerts in specification/security/resource-manager/

### CODE REVIEW

**Review by nschonni — COMMENTED** [review]
Looks like there are some tab/space indentation issues

**Comment by adxsdknet** [comment]
# Automation for azure-sdk-for-net 
A PR has been created for you: 
https://github.com/azure/azure-sdk-for-net/pull/5404 
.NET SDK Commits: 
https://github.com/adxsdknet/azure-sdk-for-net/pull/5404/commits/38e3cf4b483c8ef89d13ef71f7debda56ca1d33d

**Comment by orparnes** [comment]
@orparnes is added to the review. #Closed

File: specification/security/resource-manager/Microsoft.Security/preview/2015-06-01-preview/alerts.json [hunk 1]
```diff
@@ -561,6 +561,11 @@
           "readOnly": true,
           "type": "string",
           "description": "Azure resource ID of the workspace that the alert was reported to."
+        },
```
Comment (veronicagg) [inline specification/security/resource-manager/Microsoft.Security/preview/2015-06-01-preview/alerts.json]: minor: could you indent this property the same number of spaces as the other properties are?

File: specification/security/resource-manager/Microsoft.Security/stable/2019-01-01/alerts.json [hunk 2]
```diff
@@ -590,7 +590,12 @@
                     "readOnly": true,
                     "type": "string",
                     "description": "Azure resource ID of the workspace that the alert was reported to."
-                }
+                },
+				"correlationKey": {
```
Comment (veronicagg) [inline specification/security/resource-manager/Microsoft.Security/stable/2019-01-01/alerts.json]: minor: could you fix indenting (same comment as in the other spec)?

**Review by veronicagg — CHANGES_REQUESTED** [review]

File: specification/security/resource-manager/Microsoft.Security/preview/2015-06-01-preview/alerts.json (hunk 1 above)
Comment (t-nishab) [reply specification/security/resource-manager/Microsoft.Security/preview/2015-06-01-preview/alerts.json]: caused due to the interpretation of tabs to spaces in the ide, fixed now.

File: specification/security/resource-manager/Microsoft.Security/stable/2019-01-01/alerts.json (hunk 2 above)
Comment (t-nishab) [reply specification/security/resource-manager/Microsoft.Security/stable/2019-01-01/alerts.json]: fixed

**Review by veronicagg — APPROVED** [review]
Looks good, thanks for addressing the comments.

**Pull request merged**